pydis_site/templates/events/pages/code-jams/8/_index.html

```diff
@@ -20,18 +20,26 @@
 
     <h3 id="important-dates"><a href="#important-dates">Important Dates</a></h3>
     <ul>
-        <li>Monday, June 15 - Form to submit theme suggestions opens</li>
+        <li>Tuesday, June 15 - Form to submit theme suggestions opens</li>
         <li>Monday, June 21 - The Qualifier is released</li>
         <li>Friday, June 25 - Voting for the theme opens</li>
-        <li>Sunday, June 27 - Github Bootcamp</li>
+        <li>Sunday, June 27 - GitHub Bootcamp</li>
         <li>Wednesday, June 30 - The Qualifier closes</li>
-        <li>Friday, July 9 - Code Jam Begins</li>
+        <li>Friday, July 9 - Code Jam begins</li>
         <li>Friday, July 16 - Coding portion of the jam ends</li>
         <li>Sunday, July 18 - Code Jam submissions are closed</li>
     </ul>
+
+    <h3 id="technology"><a href="#technology">Technology</a></h3>
+    <p>
+        The chosen technology/tech stack for this year is <strong> Text User Interfaces</strong> (TUIs).
```

Comment:
```suggestion
        The chosen technology/tech stack for this year is <strong>Text User Interfaces</strong> (TUIs).
```

Reply:
I thought I checked it last night, but isn't this adding a double space now?

Reply:
No, this removes a double space.
<img width="133" alt="Screenshot 2021-06-15 at 16 05 34" src="https://user-images.githubusercontent.com/20439493/122077301-852d0800-cdf3-11eb-83d2-0425e139a5d4.png">